sentences/nb/homeassistant_HassTurnOff.yaml

```diff
@@ -2,6 +2,27 @@ language: "nb"
 intents:
   HassTurnOff:
     data:
-    - sentences:
-      - "<skru_av> <navn>"
-      - "(skru|slå) <navn> av"
+      - sentences:
+          - "<skru_av> <navn>"
+          - "(skru|slå) <navn> av"
+      - sentences:
+          - "<lukk> <navn>"
+          - "<lukk> <navn> <i_på> <område>"
+        response: cover
+      - sentences:
+          - "<lukk> garasje"
+        slots:
+          domain: cover
+          device_class: garage
+        response: cover
+      - sentences:
+          - "<lukk> [<vindu>|<gardin>|<persienne>|<skodde>] <i_på> <område>"
+          - "<lukk> <område>[s][ ][<vindu>|<gardin>|<persienne>|<skodde>]"
```

Comment:
As I wrote in another comment, shouldn't `persienne` be removed since it's part of `gardin`?

Reply:
Thank you. I have removed persienne. We might want to reintroduce rullegardin/persienne in the event we want different responses for device_class curtain vs blinds.